pkg/bitly/main.go

```diff
@@ -0,0 +1,88 @@
+package bitly
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+const SHORTENURL = "https://api-ssl.bitly.com/v4/shorten"
+
+var (
+	// std is the default instance of BitlyAPI.
+	std = New()
```

Comment:
`std` is a pointer to `BitlyAPI` also defined in this package.  Because it's a pointer to the BitlyAPI type, we can set the token on it.